src/xpu/sycl/types.hpp

```diff
@@ -30,6 +30,21 @@ namespace impl {
 namespace xpu {
 namespace sycl {
 
+#define DNNL_ARG_SRC_4 5
+#define DNNL_ARG_SRC_5 6
+#define DNNL_ARG_SRC_6 7
+#define DNNL_ARG_SRC_7 8
+#define DNNL_ARG_SRC_8 9
+#define DNNL_ARG_SRC_9 10
+#define DNNL_ARG_SRC_10 11
+#define DNNL_ARG_SRC_11 12
+#define DNNL_ARG_SRC_12 13
+#define DNNL_ARG_SRC_13 14
+#define DNNL_ARG_SRC_14 15
+#define DNNL_ARG_SRC_15 16
+
+#define MAX_NUM_TENSORS 16
```

Comment:
Is this specific to sycl sum implementation? If so I would advice to name it accordingly (e.g. DNNL_REF_SUM_MAX_NUM_TENSORS)

Reply:
done